src/main/java/org/folio/dao/lines/PoLinesPostgresDAO.java

```diff
@@ -76,4 +77,14 @@ public Future<Integer> updatePoLines(String sql, Conn conn) {
       });
     return promise.future();
   }
+
+  @Override
+  public Future<Integer> updatePoLines(String sql, DBClient dbClient) {
```

Comment:
we should not use DBClient in new code, better to use Conn entity, so you can refactor with usage of dbClient.withConn()
There are 2 reasons:

- previosly we had some issues espesially with using transactions when DBClient returns 2 different pg clients that returns different sql connections from the pool, so transaction rollback in this case will not work
- we need to write audit log for each po line update, so we need to write this audit log in scope of the same transaction that accepts Conn object